### PR TITLE
aws bookshelf links updated and unavailable links removed

### PR DESCRIPTION
**Pull request by Hafeezbaig**
# Pull Request Checklist

## First Thing

- [ ] **Review To Be Published Articles**: Use Deploy Preview (Example Deploy Preview of an older pull request is here - https://github.com/in28minutes/in28minutes.com/pull/11#issuecomment-1902625913)

## Common Mistakes
- [ ] **Date of Article**: Date of all blog articles should be yesterday or earlier
- [ ] **Notes about pictures in stories**: Make sure that all the required notes are included
- [ ] **No Typos**: Make sure that there are no typos in titles or content

## 🎨 Content Verification
- [ ] **Image Originality**: Confirm images are original
- [ ] **Text Originality**: Ensure all text is original

## ✨ Overall Quality
- [ ] **Visual Appeal**: Evaluate layout, colors, typography.
- [ ] **Style Consistency**: Check for consistent presentation style.

### CODE REVIEW

File: _blog/articles/2024-06-24-aws-bookshelf.md [hunk 1]
```diff
@@ -89,10 +89,7 @@ permalink: /aws-bookshelf
 ## S3
 
 - [Creating an S3 Bucket and Exploring the UI](https://scribehow.com/shared/Creating_an_S3_Bucket_and_Exploring_the_UI__0TFMdpbkQvS6jOisdkQjeA)
-- [Understanding S3: Objects, Buckets, and Key-Value Pairs](https://scribehow.com/shared/Understanding_S3**Objects_Buckets_and_Key_Value_Pairs**rE1C8p-eQlqBp10AbmpjzQ)
-- [How S3 Versioning Works: Deep Dive and Hands-On](https://scribehow.com/shared/How_S3_Versioning_Works__Deep_Dive_and_Hands_On__uzPEzOlzSaK8_s8MXHdAmg)
-- [Working with S3 Encryption](https://scribehow.com/shared/Working_with_S3_Encryption__MCgNQmKiTfC5SwpZBxmhgQ)
-- [Hosting Static Website in S3](https://scribehow.com/shared/Hosting_Static_Website_in_S3__aI1hLU-4Rt-FRrxZ0QOZWg)
+- [Understanding S3: Objects, Buckets, and Key-Value Pairs](https://scribehow.com/shared/Understanding_S3__Objects_Buckets_and_Key_Value_Pairs__x1GtUtZKS9axATcFzF1hTQ)
```
Comment (beena-yatin-kanyal) [inline _blog/articles/2024-06-24-aws-bookshelf.md:92]: Why did we remove too many links? are they all deleted? i was expecting updates to the URL but not deletion.

**Review by beena-yatin-kanyal — CHANGES_REQUESTED** [review]
Hi Hafeez - whenever you submit a pr please ensure following in the comment before sending it to the reviewer.
1. purpose of the pr
2. what has changed

**Review by beena-yatin-kanyal — APPROVED** [review]
approved as per the discussion.

File: _blog/articles/2024-06-24-aws-bookshelf.md (hunk 1 above)
Comment (beena-yatin-kanyal) [reply _blog/articles/2024-06-24-aws-bookshelf.md:92]: resolved.

**Comment by in28minutes** [comment]
Merged. Thank you!

**Pull request merged**